jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -122,8 +122,8 @@ function check_required_env_vars() {
 }
 
 function with_error_log() {
-  local err_log=$1
-  local cmd=("${@:2}")
+  local err_log=$1 retry_interval=$2 max_retries=$3
```

Comment:
Don't mix retrying functionality into function `with_error_log`, but implement retrying as an own function.

Reply:
In that case the separate function would have some lines of code in common with `with_error_log`. Would it be fine?

Reply:
Yes, this would be fine. But I don't see what exactly would be duplicated, except the boilerplate like definition of local variables.

Reply:
I took them now into separate functions.